build.gradle.kts

```diff
@@ -227,6 +220,11 @@ val functionalTest by tasks.registering(Test::class) {
   beforeTest(closureOf<TestDescriptor> {
     logger.lifecycle("Running test: $this")
   })
+
+  // TODO cleanup: do only for AGP tests?
```

Comment:
TODO

Reply:
I think I'll tackle this question in a separate PR. If I want to run the JVM tests against JDK 11 and the Android tests against JDK 17, I'll need separate test tasks, or test filters and separate invocations, or something like that.